src/commands/doctor.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as os from 'os';
+import { exec } from 'child_process';
+import { flags, SfdxCommand } from '@salesforce/command';
+import { Messages, Lifecycle } from '@salesforce/core';
+import { Doctor as SFDoctor, SfDoctorDiagnosis } from '../doctor';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-info', 'doctor');
+
+export default class Doctor extends SfdxCommand {
+  public static description = messages.getMessage('commandDescription');
+
+  public static examples = messages.getMessage('examples').split(os.EOL);
+
+  // Hide for now
+  public static hidden = true;
```

Comment:
Until the command has more features added and has some tests I think we should hide it.